frontend/src/scenes/trends/ActionFilter/ActionFilter.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react'
+import { useActions, useValues } from 'kea'
+import { entityFilterLogic } from './actionFilterLogic'
+import { ActionFilterRow } from './ActionFilterRow'
+
+export function ActionFilter(props) {
+    const { allFilters, filters } = useValues(entityFilterLogic)
+    const { createNewFilter, initializeLocalFilters } = useActions(entityFilterLogic)
+
+    useEffect(() => {
```

Comment:
edge case where actionfilterlogic is waiting on trendlogic to provide an accurate copy of filters from params because actionfilterlogic eventually handles the filters